ttsai/rotator_verification.ipynb

```diff
@@ -0,0 +1,3831 @@
+{
```

Comment:
<p>Here starts test 4.9 Slewing supersession. </p><p>Test looks good.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/72/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='83'/>

Reply:
It is not easy to fine-tune that position. The existed test case can test the slewing supersession already. Thanks!